lib/charms/nrpe_exporter/v0/nrpe_exporter.py

```diff
@@ -453,21 +453,25 @@ def _generate_data(self, relation) -> Tuple[list, list]:
 
     def _generate_alert(self, relation, cmd, id, unit) -> dict:
         """Generate an on-the-fly Alert rule."""
+        pat = r"^(.*?)[-_](\d+)$"
+        if match := re.match(pat, id.replace("_", "-")):
```

Comment:
**nitpick (non-blocking)**: how about `pattern` instead of `pat`?

Reply:
Done.